src/components/Form.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <form @submit.prevent="submitDetails">
+    <div class="flex flex-col gap-y-10 flex-nowrap text-base w-full">
+      <RadioButtons v-model="value" :icons="icons" />
+      <div>
+        <p class="font-bold">Punkt początkowy:</p>
+        <TextInput v-model:value="address"></TextInput>
```

Comment:
a nie :value?

Reply:
jesli props to modelValue to wystarczy samo v-model jesli props jest value to v-model:value